gtubt/lib/ui/pages/profile_page.dart

```diff
@@ -242,6 +245,7 @@ class _ProfilePageState extends State<ProfilePage> {
   }
 
   Widget _phoneNumber(String? phone) {
+    // TODO: Needs custom formatting, we should use one of phone input widgets.
```

Comment:
could you open issue card for this todo

Reply:
https://trello.com/c/cnFVbgRx/286-telefon-input-widget-i-eklenecek-profil-edit